modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/Main.scala

```diff
@@ -142,70 +44,16 @@ object Main {
           Await.result(as.terminate(), 10 seconds)
       }
 
-      val instanceIndexingSettings = InstanceIndexingSettings(
-        settings.Sparql.Instances.Index,
-        apiUri,
-        settings.Sparql.Instances.GraphBaseNamespace,
-        settings.Prefixes.CoreVocabulary)
-
-      SequentialIndexer.start[InstanceEvent](
-        InstanceIndexer[Future](sparqlClient, instanceIndexingSettings).apply,
-        "instances-to-3s",
-        settings.Persistence.QueryJournalPlugin,
-        "instance",
-        "sequential-instance-indexer")
-
-      val schemaIndexingSettings = SchemaIndexingSettings(
-        settings.Sparql.Schemas.Index,
-        apiUri,
-        settings.Sparql.Schemas.GraphBaseNamespace,
-        settings.Prefixes.CoreVocabulary)
-
-      SequentialIndexer.start[SchemaEvent](
-        SchemaIndexer[Future](sparqlClient, schemaIndexingSettings).apply,
-        "schemas-to-3s",
-        settings.Persistence.QueryJournalPlugin,
-        "schema",
-        "sequential-schema-indexer")
-
-      val domainIndexingSettings = DomainIndexingSettings(
-        settings.Sparql.Domains.Index,
-        apiUri,
-        settings.Sparql.Domains.GraphBaseNamespace,
-        settings.Prefixes.CoreVocabulary)
-
-      SequentialIndexer.start[DomainEvent](
-        DomainIndexer[Future](sparqlClient, domainIndexingSettings).apply,
-        "domains-to-3s",
-        settings.Persistence.QueryJournalPlugin,
-        "domain",
-        "sequential-domain-indexer")
-
-      val orgIndexingSettings = OrganizationIndexingSettings(
-        settings.Sparql.Organizations.Index,
-        apiUri,
-        settings.Sparql.Organizations.GraphBaseNamespace,
-        settings.Prefixes.CoreVocabulary)
-
-      SequentialIndexer.start[OrgEvent](
-        OrganizationIndexer[Future](sparqlClient, orgIndexingSettings).apply,
-        "organization-to-3s",
-        settings.Persistence.QueryJournalPlugin,
-        "organization",
-        "sequential-organization-indexer")
+      BootstrapIndexing.startIndexing(settings, bootstrap.sparqlClient, bootstrap.apiUri)
     }
 
-    val provided = settings.Cluster.Seeds
-      .map(addr => AddressFromURIString(s"akka.tcp://${settings.Description.ActorSystemName}@$addr"))
-    val seeds = if (provided.isEmpty) Set(cluster.selfAddress) else provided
-
-    cluster.joinSeedNodes(seeds.toList)
-
+    bootstrap.cluster.joinSeedNodes(bootstrap.seeds.toList)
```

Comment:
Perhaps this could be a method in a  `BootstrapSevice` like  `joinCluster`, it feels a bit weird to pass values stored within a `BootstrapSevice` object to another object within  `BootstrapSevice`